local_planner/src/nodes/local_planner.cpp

```diff
@@ -404,7 +401,10 @@ void LocalPlanner::updateObstacleDistanceMsg() {
 void LocalPlanner::reprojectPoints(Histogram histogram) {
   double n_points = 0;
   double dist, age;
+  float half_res = ALPHA_RES / 2;
```

Comment:
are you aware of what this does? Careful...!

Reply:
half_res needs to be an integer. Otherwise it looks fine to me. The resolution is hard coded and it needs to be dividable by two, otherwise we have a lot more problems. I will write a test for valid resolutions in another PR.